src/content/pages/sprints.mdx

```diff
@@ -143,6 +143,16 @@ and tag us @EuroPython ([twitter](https://twitter.com/europython),
 - Contact person: Vytautas Liuolia ([@vytas7](https://github.com/vytas7))
 - Links: https://github.com/falconry/falcon
 
+### OpenTelemetry: improve an instrumentation, add OpenTelemetry support to 3rd party libraries
```

Comment:
If you don't mind, we'll stick to project names in the title, otherwise the web page will have huge sentences with a huge font 🤪 

```suggestion
### OpenTelemetry
```

One thing that I've found works really well is creating a “meta-issue” in the project's repo and linking prominently to that in the links section below.